From 96da404c299760e4a9048a6e72d93e51cac3cad9 Mon Sep 17 00:00:00 2001
From: pwielders <pierre@wielders.net>
Date: Thu, 11 Mar 2021 11:21:50 +0100
Subject: [PATCH 1/2] Merge pull request #496 from
 npoltorapavlo/RDK-29971_thunderfix
Source: COMCAST
License: GPLV2
Upstream-Status: Pending

RDK-29971 : Fix JSONRPCSupportsEventStatus
Signed-off-by: Nikita Poltorapavlo <npoltorapavlo@productengine.com>
---
 Source/plugins/JSONRPC.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/plugins/JSONRPC.h b/Source/plugins/JSONRPC.h
index 04ab668d..cd3c2a5d 100644
--- a/Source/plugins/JSONRPC.h
+++ b/Source/plugins/JSONRPC.h
@@ -460,7 +460,7 @@ namespace PluginHost {
         virtual void Unsubscribe(Core::JSONRPC::Handler& handler, const uint32_t channelId, const string& eventName, const string& callsign, Core::JSONRPC::Message& response)
         {
             NotifyObservers(eventName, callsign, Status::unregistered);
-            JSONRPC::Subscribe(handler, channelId, eventName, callsign, response);
+            JSONRPC::Unsubscribe(handler, channelId, eventName, callsign, response);
         }
 
     private:
-- 
2.29.0

